pkg/storage/minio/minio.go

```diff
@@ -39,8 +39,9 @@ func NewStorage(conf *config.MinioConfig, timeout time.Duration) (storage.Backen
 	err = minioClient.MakeBucket(bucketName, region)
 	if err != nil {
 		// Check to see if we already own this bucket
-		exists, err := minioClient.BucketExists(bucketName)
-		if err == nil && !exists {
+		exists, _ := minioClient.BucketExists(bucketName)
+		if !exists {
```

Comment:
@jamesnaftel should we check `err` after this check, in case there was something else? Or will an error always be `nil` if `exists` is true?

Reply:
In the current minio-go client, true only gets return with a nil.  We could chain the BuckExists error in the failure case with the MakeBucket error but it looks like we would usually get the same error.

[minio-go: BucketExists](https://github.com/minio/minio-go/blob/26dec1a6ef2e943908b058233dbd091b411696d5/api-stat.go#L31)

Reply:
Got it, that makes sense. Thanks for explaining @jamesnaftel !